### PR TITLE
orca-5: Remove poretools - brewsci/homebrew-bio#266

### PR DESCRIPTION
**Pull request by suujia**
remove from orca-5, versions.tsv, and singularity.1 

```
Porecamp: Primary purpose of converting FAST5 to FASTQ files is no longer needed. Albacore does that. NanoPlot can be used for plotting and reporting https://github.com/wdecoster/NanoPlot
It seems to me that we can remove poretools.
```
-sjackman

https://github.com/brewsci/homebrew-bio/pull/266

### CODE REVIEW

File: orca-5/Dockerfile [hunk 1]
```diff
@@ -62,8 +62,8 @@ pixman \
 pkg-config \
 plink \
 poa \
-poretools \
 prodigal \
 prokka \
 proteinortho \
 psmc
+
```
Comment (sjackman) [inline orca-5/Dockerfile]: Please remove this blank line.

File: versions.tsv [hunk 2]
```diff
@@ -334,7 +334,6 @@ pixman	0.34.0_1
 pkg-config	0.29.2_1
 plink	1.07
 poa	2.0
-poretools	0.6.0_6
```
Comment (sjackman) [inline versions.tsv]: This file `versions.tsv` is auto-generated. Please remove this line. When we're ready to release a new version of ORCA, I'll show you how to update this file. Thanks for being so thorough!

**Comment by sjackman** [comment]
Can you please link to the original brewsci/bio PR, and include in the commit log the error message and reason that poretools is being removed from ORCA?

**Comment by sjackman** [comment]
Super confusing, but it looks like there are two similarly named tools: `dockerfilelint` and `dockerfile_lint`. We're currently using the former, I believe, though the latter seems more popular. I have no preference. We could switch to the latter if that helps.
See https://www.npmjs.com/package/dockerfile_lint
and https://www.npmjs.com/package/dockerfilelint
and this very helpful blog post: https://jacace.wordpress.com/2016/12/01/tools-to-lint-dockerfiles/

**Comment by suujia** [comment]
Oh I see. Thanks for the links! I will look into it :P

**Comment by sjackman** [comment]
I've opened an issue at https://github.com/projectatomic/dockerfile_lint/issues/110 to ask how to override the severity of a default rule.

**Comment by sjackman** [comment]
Could you please open a new separate PR to change from using `dockerfilelint` to `dockerfile_lint`? This PR can then be changed back to its original purpose of removing poretools.

**Comment by suujia** [comment]
haha sorry for the spam. I will work on that. I have squashed it (and merged) only the poretools. I will make another pull request for dockerfile_lint

**Comment by sjackman** [comment]
No worries. Thanks for making this fix, Susie.

**Pull request merged**